core/storage/job/gae_models.py

```diff
@@ -81,11 +110,30 @@ def get_new_id(cls, entity_name):
 
     @property
     def is_cancelable(self):
+        """Determines if the running job can be canceled.
```

Comment:
Whether the job can be canceled.

(We can't assume that it's currently running.)